e2e/tests/policy-forms.spec.ts

```
@@ -481,16 +481,286 @@ test.describe('other policy create pages render', () => {
     await expect(page.locator('.monaco-editor').first()).toBeVisible({ timeout: 15_000 });
   });
 
-  test('TokenRateLimitPolicy create page renders YAML editor', async ({ page }) => {
+  test('TokenRateLimitPolicy create form renders with required fields', { tag: '@smoke' }, async ({ page }) => {
```

ESLint

[error] Replace `'TokenRateLimitPolicy·create·form·renders·with·required·fields',·{·tag:·'`@smoke`'·},` with `⏎····'TokenRateLimitPolicy·create·form·renders·with·required·fields',⏎····{·tag:·'`@smoke`'·},⏎···`

(prettier/prettier)